src/middlewares/mail.js

```diff
@@ -14,14 +15,16 @@ import Mail from '../services/mail/Mail';
  */
 export const SendVerificationEmail = async (req, res, next) => {
   let { email, firstName, lastName } = req.body;
+  console.log(email)
   email = email ? email.trim() : '';
   firstName = firstName ? firstName.trim() : '';
   lastName = lastName ? lastName.trim() : '';
   /**
    * @var {id} id is the user unique id from Table column
    */
-  const id = 'some_encoded_identiity';
+  const id = req.user ? req.user.email : 'some_encoded_identity';
```

Comment:
What do you mean by some encoded entity?

Reply:
It has been fixed